.circleci/config.yml

```diff
@@ -85,6 +85,11 @@ install_dep: &install_dep
 
```

Comment:
thanks for all the CI changes, LGTM and pretty useful

Reply:
> Have you run into any numerical instability issues during training? Or, have you done any checks as was done

hi, @lucidrains , do you meet such precision loss situation during your training? I meet it when train SWIN-T.